flink-core/src/main/java/org/apache/flink/configuration/ConfigConstants.java

```diff
@@ -1233,6 +1239,9 @@
 	/** ZooKeeper default leader port. */
 	public static final int DEFAULT_ZOOKEEPER_LEADER_PORT = 3888;
 
+	/** Defaults for ZK client security **/
+	public static final boolean DEFAULT_ZOOKEEPER_SASL_DISABLE = true;
```

Comment:
I'm wondering, shouldn't the default be true? If I configure Jaas, then I probably want to connect to a secured Zookeeper by default.